File: 1주차/PGS_같은 숫자는 싫어/seoheun.js
function solution(arr) {
    var answer = [];

    for (let i = 0; i < arr.length; i++) {
        if (answer.length === 0) {
            answer.push(arr[i]);
        } else {
            if (arr[i] !== answer[answer.length - 1]) {
                answer.push(arr[i]);
            }
        }
    }

    return answer;
}

let start = new Date(); // 시작
solution([1, 1, 3, 3, 0, 1, 1]);
let end = new Date(); // 종료

console.log(end - start); // 경과시간 (밀리초)

File: 2주차/PGS_가장 큰 수/seoheun.py
# 답 참고 풀이

def solution(numbers):
    arr = []
    for num in numbers:
        arr.append(str(num))
        
    arr.sort(key = lambda x: x * 3, reverse = True)
    
    return str(int(''.join(arr))) # 00 을 0 으로